RFS/src/main/java/com/rfs/common/http/AuthTransformer.java

```diff
@@ -0,0 +1,39 @@
+package com.rfs.common.http;
+
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import reactor.core.publisher.Mono;
+
+public interface AuthTransformer {
+    Mono<TransformedRequest> transform(String method, String path, Map<String, List<String>> headers, Mono<String> body);
```

Comment:
```suggestion
    Mono<TransformedRequest> applyAuth(String method, String path, Map<String, List<String>> headers, Mono<String> body);
```

Reply:
Renamed AuthTransformer to RequestTransformer instead